hal/ia32/cpu.c

```diff
@@ -161,54 +175,60 @@ int hal_cpuReschedule(spinlock_t *spinlock, spinlock_ctx_t *scp)
 			spinlock->dmin = spinlock->e - spinlock->b;
 		}
 
-		__asm__ volatile(
-			"xorl %%eax, %%eax; "
-			"incl %%eax; "
-			"xchgl %0, %%eax; "
-		:: "m" (spinlock->lock)
+		/* clang-format off */
+		__asm__ volatile (
+			"xorl %%eax, %%eax\n\t"
+			"incl %%eax\n\t"
+			"xchgl %0, %%eax\n\t"
```

Comment:
Redundant `\n\t`, last delimiter is generated by `gcc` always.
```suggestion
			"xchgl %0, %%eax"
```